src/components/my-app.js

```diff
@@ -10,27 +10,32 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 
 import { LitElement, html } from '@polymer/lit-element';
 import { setPassiveTouchGestures } from '@polymer/polymer/lib/utils/settings.js';
-
-// These are the helpers needed by this element.
-import { installMediaQueryWatcher } from 'pwa-helpers/media-query.js';
-import { installOfflineWatcher } from 'pwa-helpers/network.js';
-import { installRouter } from 'pwa-helpers/router.js';
-import { updateMetadata } from 'pwa-helpers/metadata.js';
+import {
```

Comment:
I'm not sure this is a good thing to do. Even if in this way the import is more elegant and it serves functionally the same purpose, it might increase the bundle size. Let me explain better:

With
```
import { connect } from 'pwa-helpers/connect-mixin.js';
```
You are telling: "Get the `pwa-helpers/connect-mixin.js` module and use `connect` contained in it"

While using
```
import { connect } from 'pwa-helpers';
```
You are telling: "Get the `pwa-helpers` module (that includes **all** the various `pwa-helpers`) and use only `connect` from it"
So you are most probably adding to the bundle some helper that isn't needed (nor used) at all

Of course in this case this is not a problem, since we are using all the exports from `pwa-helpers`, but with bigger libraries this can be a **real** problem (take a look at [this example](https://file-wkbcnlcvbn.now.sh/import-cost.gif)), so I think it would be a better practice to always use the `library/submodule` syntax to favor tree shaking

Reply:
Tree shaking, by definition, means removing exports from bundle that are not used. And as long as import expression is statically analysable it is treeshakable:)
 
Your statement is true if import will be:
```javascript
import * as pwaHelpers from 'pwa-helpers';
```

Reply:
Thank you for clarifying, you are definitely right! The way you imported `pwa-helpers` is much better than the old way then

Reply:
@frankiefu do you know if it will take a long time to arrive?

Keeping in mind that all helpers are used, maybe it would be better to leave it that way? WDYT?

Reply:
I expect it might take a bit of time given other issues in tools. Here is the [issue](https://github.com/Polymer/tools/issues/396).
It's true that the default starter template is currently using all the helpers. But some users may not need all the helpers so we should make that works too.